Basic Rules.txt

```diff
@@ -213,29 +213,31 @@ rolls are Perception and Dexterity. The Perception stat affects a ranged weapon'
 bracket miss chance, and the Dexterity stat affects melee miss chance.
 
 COMBAT MISS CHANCE BONUSES TABLE
-╔════════╦═════════════╗		
-║ Stat   ║ Stat Bonus  ║
-╠════════╬═════════════╣		
-║ 1      ║ -1.5	       ║	
-╠════════╬═════════════╣		
-║ 2      ║ -1	       ║	
-╠════════╬═════════════╣		
-║ 3-4    ║ -0.5	       ║		
-╠════════╬═════════════╣	
-║ 5      ║  0	       ║	
-╠════════╬═════════════╣	
-║ 6-7    ║  0.5	       ║	
-╠════════╬═════════════╣	
-║ 8-9    ║  1	       ║	
-╠════════╬═════════════╣	
-║ 10-11  ║  1.5	       ║	
-╠════════╬═════════════╣	
-║ 12-13  ║  2	       ║	
-╠════════╬═════════════╣	
-║ 14     ║  2.5	       ║		
-╠════════╬═════════════╣
-║ 15     ║  3	       ║		
-╚════════╩═════════════╝
+
++--------+-------------+		
+| Stat   | Stat Bonus  |
++--------+-------------+		
+| 1      | -1.5	       |	
++--------+-------------+		
+| 2      | -1	       |	
++--------+-------------+		
+| 3-4    | -0.5	       |		
++--------+-------------+	
+| 5      |  0	       |	
++--------+-------------+	
+| 6-7    |  0.5	       |	
++--------+-------------+	
+| 8-9    |  1	       |	
++--------+-------------+	
+| 10-11  |  1.5	       |	
++--------+-------------+	
+| 12-13  |  2	       |	
++--------+-------------+	
+| 14     |  2.5	       |		
++--------+-------------+
+| 15     |  3	       |		
++--------+-------------+
+

```

Comment:
Something strange happened to the trailing pipes here.

Reply:
It be proper to my eye, sir?
![it be proper to my eye](https://user-images.githubusercontent.com/13957025/34425176-74a1e794-ebde-11e7-8e02-d81878db808c.png)

Reply:
Iiiiinteresting. Here's what it looks like on my screen.
![image](https://user-images.githubusercontent.com/23040366/34427219-a68c952c-ebf4-11e7-9a65-ae2a54fcbf97.png)

Reply:
It's just an artifact of the changes view. I went to the raw file and it's perfectly normal there.
Disregard.